server.js

```diff
@@ -104,8 +81,8 @@ http.createServer((req, res) => {
   if (req.url === '/') {
     req.url = '/en'
   }
-  mount(req, res, redirectToEnglishUrl(req, res))
+  mount(req, res)
 }).listen(port, () => console.log(`\x1B[32mServer running at http://localhost:${port}/en/\x1B[39m`))
```

Comment:
Console output should be `http://localhost:${port}/${selectedLocales[0]}/`, I guess.

Reply:
Thank you for your review. I'm about to leave on holiday with the family starting today. We'll be disconnected until the 16th, so i won't be available during this period . Hope it's ok. Hopefully you'll be able to discuss about build times, or if we should try a different approach.
Thanks again. 👍 👍